blocks/article-header/article-header.css

```diff
@@ -0,0 +1,19 @@
+.block .article-byline p {
+  display: inline;
+  margin-right: 20px;
+  font-size: 14px;
+  font-weight: 400;
+}
```

Comment:
Now that this CSS is in the block we can remove https://github.com/hlxsites/servicenow/blob/main/styles/styles.css#L399-L402

But please use `rem` for margin and font size to keep it consistent with the customer's original CSS